src/overlays/actors/ovl_En_Wood02/z_en_wood02.c

```diff
@@ -214,11 +214,11 @@ void EnWood02_Init(Actor* thisx, PlayState* play) {
         case WOOD_BUSH_GREEN_LARGE_SPAWNER:
         case WOOD_BUSH_BLACK_LARGE_SPAWNER:
             spawnType = 1;
-
+            // fallthrough
```

Comment:
Could we use the `FALLTHROUGH` attribute instead? You may need to include the `attributes.h` header tho
```suggestion
            FALLTHROUGH;
```

Reply:
We could, but I'm happy to make a separate PR afterwards using the `FALLTHROUGH` attribute everywhere?